src/natcap/invest/utils.py

```diff
@@ -484,10 +484,8 @@ def build_lookup_from_csv(
     """
     # Check if the file encoding is UTF-8 BOM first
     encoding = None
-    with open(table_path, 'rb') as file_obj:
-        first_line = file_obj.readline()
-        if first_line.startswith(codecs.BOM_UTF8):
-            encoding = 'utf-8-sig'
+    if has_utf8_bom(table_path):
+        encoding = 'utf-8-sig'
```

Comment:
Just looking at the `has_utf8_bom` here and below for the dataframe. I thought Pandas had a way where you could export a dataframe as a dictionary? I'm wondering if there's duplicate code here such that `build_lookup_from_csv` could be implemented from a call to `read_csv_to_dataframe` which handles all the weird CSV formatting an encodings?

Reply:
Yes indeed!  Updated in bb8dc54c.